src/test/miner_tests.cpp

```diff
@@ -553,15 +601,14 @@ void MinerTestingSetup::TestPrioritisedMining(const CChainParams& chainparams, c
 BOOST_AUTO_TEST_CASE(CreateNewBlock_validity)
 {
     // Note that by default, these tests run with size accounting enabled.
-    const auto chainParams = CreateChainParams(*m_node.args, CBaseChainParams::MAIN);
-    const CChainParams& chainparams = *chainParams;
     CScript scriptPubKey = CScript() << ParseHex("04678afdb0fe5548271967f1a67130b7105cd6a828e03909a67962e0ea1f61deb649f6bc3f4cef38c4f35504e51ec112de5c384df7ba0b8d578a4c702b6bf11d5f") << OP_CHECKSIG;
     std::unique_ptr<CBlockTemplate> pblocktemplate;
 
     fCheckpointsEnabled = false;
 
+    CTxMemPool& tx_mempool{*m_node.mempool};
```

Comment:
Question: why doesn't this end up as `CTxMemPool& tx_mempool{MakeMempool()};` ?

Reply:
I don't know. There shouldn't be any difference. Happy to add a commit to change it.

Reply:
There is only one test case in this file and this is the first use of the mempool, so we can use the existing one from the testing setup, or a freshly made one. Both should be "clean", so either can be used. Maybe someone can flip a coin?